cypress/integration/models/keycloak/users/user.ts

```diff
@@ -26,15 +26,18 @@ export class User {
 
     static keycloakUrl = tackleUiUrl + "/auth/";
 
-    static loginAdmin(): void {
+    static loginAdmin(loggedIn = false): void {
```

Comment:
Please change the name to loginKeycloackAmin().
It is confusing otherwise

Reply:
Done